todoman/cli.py

```diff
@@ -224,49 +247,48 @@ def edit(ctx, id, todo_properties, interactive):
             setattr(todo, key, value)
 
     if interactive or (not changes and interactive is None):
-        ui = TodoEditor(todo, ctx.obj['db'].lists(), ctx.obj['formatter'])
+        ui = TodoEditor(todo, ctx.db.lists(), ctx.ui_formatter)
```

Comment:
I think the mix between interactive and non-interactive *only* makes sense using `--porcelain`. Usecase is a script that launches `todo new` with certain default values. `todo edit` works the same way, but with more default values.